### PR TITLE
Inetutils => 2.4

### PR DESCRIPTION
**Pull request by satmandu**
- Also fixup `iputils` filelists.
- Update `alive` with preflight section

Works properly:
- [x] `x86_64`
- [x] `i686`
- [x] `armv7l` 

### Run the following to get this pull request's changes locally for testing.
```
CREW_REPO=https://github.com/satmandu/chromebrew.git CREW_BRANCH=inetutils crew update
```

### CODE REVIEW

File: packages/alive.rb [hunk 1]
```diff
@@ -16,4 +16,11 @@ class Alive < Autotools
 
   depends_on 'guile' # R
   depends_on 'inetutils' # L
+
+  def self.preflight
+    puts "You need to be in the VT-2 shell for the install of alive to work.".lightblue
+    puts "Make sure you have used Ctrl-Alt-{F2/Right arrow/Refresh} to switch to VT-2.".lightblue
+    puts "Then login as 'chronos' and install alive with 'crew install alive'.".lightblue
+    puts "You can then switch back to your regular Chromeos windows with Ctrl-Alt-{F1/Left arrow}.".lightblue
```
Comment (uberhacker) [inline packages/alive.rb:24]: Why is this necessary?

Comment (satmandu) [reply packages/alive.rb:24]: The configure script calls ping from inetutils, and that only works with sudo now, so configure needs to be run in VT2 where sudo works...

**Review by uberhacker — APPROVED** [review]
